docs/google_analytics_custom_import.md

```diff
@@ -0,0 +1,23 @@
+# Authorisation for Google Analytics custom import
+
+1. Generate a service account (see [Google Documentation](https://developers.google.com/api-client-library/ruby/auth/service-accounts)) and store the JSON key somewhere safe.
+2. Extract the `client_email` value from the JSON key and make it available to the app in the "GOOGLE_CLIENT_EMAIL" environment variable.
+3. Extract the `private_key` value from the JSON key and make it available to the app in the "GOOGLE_PRIVATE_KEY" environment variable. 
+4. Make sure your local_env.yml file contains an entry for: 
+`GOOGLE_ACCOUNT_TYPE: 'service_account'`
```

Comment:
`local_env.yml` is only used in dev mode, but from reading the google lib docs it looks like we'll need the `GOOGLE_ACCOUNT_TYPE` env var set properly for all envs.  What's our plan for production, and should we mention it here?  Maybe we can just assume it's always a 'service_account' and explicitly set the env var to that in code?

Reply:
I've set the ENV variable for `GOOGLE_ACOUNT_TYPE` in the code, so it will always be set to `service_account` before we attempt authentication. 

I'm going to add the other five environment variables to `govuk_secrets`:
`GOOGLE_CLIENT_EMAIL`
`GOOGLE_PRIVATE_KEY`
`GOOGLE_EXPORT_ACCOUNT_ID`
`GOOGLE_EXPORT_CUSTOM_DATA_IMPORT_SOURCE_ID`
`GOOGLE_EXPORT_TRACKER_ID`